mockgcp/common/operations/operations.go

```diff
@@ -100,6 +110,8 @@ func (s *Operations) StartLRO(ctx context.Context, callback func() (proto.Messag
 				klog.Warningf("error building anypb for result: %v", err)
 				finished.Result = &pb.Operation_Response{}
 			} else {
+				rewriteTypes(resultAny)
```

Comment:
Do we have any concern about the paths where this isn't called? If we fail to cast and can't fix the url, would it make sense to put an error in the response status as a bread crumb?

Reply:
It is a mock, so we could panic....